src/civic/index.js

```diff
@@ -462,100 +496,78 @@ const processEvidenceRecord = async (opt) => {
  * Get a list of CIViC Evidence Items which have since been deleted.
  * Returns the list of evidence item IDs to be purged from GraphKB
  *
- * @param {string} baseUrl base url for the CIViC API
```

Comment:
why did you remove the variable name here?

Reply:
Has been changed by '@param {string} url endpoint for the CIViC API' on line 499.

Reply:
Ah, yup. This was my mistake. I misread the diff here